system/aws-ecr-creds-helper/templates/cronjob.yaml

```diff
@@ -0,0 +1,51 @@
+apiVersion: batch/v1beta1
+kind: CronJob
+metadata:
+  name: aws-ecr-creds-helper
+spec:
+  concurrencyPolicy: Forbid
+  jobTemplate:
+    metadata:
+      name: ecr-creds-helper
+    spec:
+      backoffLimit: 3
+      template:
+        metadata:
+          name: ecr-creds-helper
```

Comment:
If we add a label to the pod spec (e.g. `cronjob=aws-ecr-creds-helper`) we can query more easily in prometheus. if we do our own alert rule we might as well make it easier for us

Reply:
hmm or maybe this needs to go as a label on the `jobTemplate` above...

Reply:
I will keep them added for the sake of completeness although as we found out they still do not show up under job in Prometheus